src/test/java/org/fundacionjala/coding/MarcoM_spin_word/TestEAN.java

```diff
@@ -0,0 +1,32 @@
+package org.fundacionjala.coding.MarcoM_spin_word;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+
+/**
+ * Created by Administrator on 5/26/2017.
+ */
+public class TestEAN {
+
+    /**
+     * verify.
+     */
+    @Test
+    public void validateTrue() {
+        EAN validator = new EAN();
```

Comment:
You are repeating this code in both tests
Please put them in @before method or convert it to a static method.